modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,425 @@
+                    "${IDP}:sub": "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster"
+                }
+            }
+        }
+    ]
+}
+EOF
+----
++
+.. Create and verify the role with the generated trusted policy:
++
+[source, terminal]
+----
+$ aws iam create-role --role-name albo-controller --assume-role-policy-document file://albo-controller-trusted-policy.json
+CONTROLLER_ROLE_ARN=$(aws iam get-role --role-name albo-controller | \grep '^ROLE' | \grep -Po 'arn:aws:iam[0-9a-z/:\-_]+')
+echo $CONTROLLER_ROLE_ARN
+----
++
+.. Attach the controller's permission policy to the role:
++
+[source,terminal]
+----
+$ curl -o albo-controller-permission-policy.json https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/assets/iam-policy.json
+aws iam put-role-policy --role-name albo-controller --policy-name perms-policy-albo-controller --policy-document file://albo-controller-permission-policy.json
+----
++
+.. Generate the operator's aws credentials:
++
+[source,terminal]
+----
+$ cat <<EOF > albo-controller-aws-credentials.cfg
+[default]
+sts_regional_endpoints = regional
+role_arn = ${CONTROLLER_ROLE_ARN}
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+----
++
+.. Create the operator's credentials secret with the generated aws credentials
++
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator create secret generic aws-load-balancer-controller-cluster --from-file=credentials=albo-controller-aws-credentials.cfg
+----
++
+. Create an AWS IAM role and assign the ALBO and ALBC AWS IAM p.olicies as defined in the previous steps. For more information on creating AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html[Creating IAM roles]. For more information on adding AWS IAM policies to AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html[Adding and removing IAM identity permissions].
+
+. Assign the following trust policy to the newly-created AWS IAM role. For more information on assigning trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
+
++
+--
+[source,json]
+----
+{
+"Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "arn:aws:iam::{AWS-AccountNo}:{Cluster's OIDC Endpoint}" <1>
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringEquals": {
+                    "{Cluster's OIDC Endpoint}:sub": [
+                        "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager",
+                        "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-cluster"
+                    ]
```

Comment:
This trust policy is not required anymore since in previous steps we already created two separated policies for `aws-load-balancer-operator-controller-manager` and `aws-load-balancer-controller-cluster`

Reply:
Waiting on response for clarification.